src/utils/env.ts

```diff
@@ -109,6 +109,7 @@ export function readEnvironmentConfig(
   // Read from home dir
   const homedirEnvFile = join(homedir(), ENV_FILE_NAME);
   if (existsSync(homedirEnvFile)) {
+    console.log('here!!!!');
```

Comment:
🔍

Reply:
This is an artifact of me trying to debug why travis was failing (see my thread w Billy in #discuss-javascript - seems like there might be an actual bug in either travis or jest). That's why I hadn't actually requested a review yet - plan was to debug that then get rid of that last commit `testing the test`, which introduced this. I should have been more communicative on that point, though, sorry.

Reply:
Ok sure, feel free to request another review when everything is ready.